Firestore/core/src/firebase/firestore/model/field_value.h

```diff
@@ -128,6 +133,32 @@ class FieldValue {
     return ObjectValue{object_value_};
   }
 
+  /**
+   * Returns a FieldValue with the field at the named path set to value.
+   *
+   * @param field_path The field path to set.
+   * @param value The value to set.
+   * @return A new FieldValue with the field set.
+   */
+  FieldValue Set(FieldPath field_path, FieldValue value) const;
+
+  /**
+   * Returns a FieldValue with the field path deleted. If there is no field at
+   * the specified path nothing is changed.
```

Comment:
But the `FieldValue` still gets copied, right?

Reply:
Yes, that's true. Updated docs.